Build/ansible/playbooks/ubuntu_14_16_x86.yml

```diff
@@ -135,6 +135,39 @@
             job="/usr/bin/apt-get update && /usr/bin/apt-get -y upgrade"
             state=present
 
+    - name: Install Test tools
+      apt: name={{item}} state=installed
+      with_items:
+        - ant
+        - ant-contrib
+        - git
```

Comment:
hmm so wget and git should be added to the build tools area as they are also required there